### ISSUE TITLE
About Table 5 evaluation in the paper

### ISSUE THREAD

**Issue by PointsCoder** [flymin/MagicDrive3D#5]
Thanks for your great work.

I have one question about the evaluation on perception support. What does no-rig mean in Table 5, and how did you generate those data? Thank you.

![image](https://github.com/user-attachments/assets/0d39fdf4-592b-429e-9fbb-c169dcb3004c)

**Comment by flymin** [comment]
Hi, we use the nuScenes dataset. "no rig" means we render with the camera parameters from the dataset. Other rigs are transformations upon the original camera parameters from the dataset.

**Comment by PointsCoder** [comment]
Thanks for your reply! 

If no rig is the original camera parameter, should the values be the same as in Table 3 of the MagicDrive? Correct me if I was wrong.

Another question is how do you generate real images (1st line) with depth +0.5, ... etc.?

**Comment by flymin** [comment]
No, all the testing data in this table are rendered (or generated) using our method. Since we convert each scene annotation to be static over time (originally there are dynamic objects), the annotation of this rendered test set is different from the nuScenes validation set.

The second column shows differences in model training data, where "only real data" means we only use the official training split of nuScenes (i.e., standard CVT training). Only lines for "w/ random aug. of 4 rigs" have rigged data for CVT training.
